### PR TITLE
Added 08-2.md and the summary

### PR DESCRIPTION
**Pull request by kawshik8**

### CODE REVIEW

File: docs/en/week08/08-2.md [hunk 1]
```diff
@@ -0,0 +1,164 @@
+---
+lang-ref: ch.08-2
+title: 
+authors: Henry Steinitz, Rutvi Malaviya, Aathira Manoj
+date: 23 Mar 2020
+---
+
+# Energy-based Models, Part II
```
Comment (Atcold) [inline docs/en/week08/08-2.md]:
```suggestion
title: Energy-based Models, Part II
authors: Henry Steinitz, Rutvi Malaviya, Aathira Manoj
date: 23 Mar 2020
---

```

File: docs/en/week08/08-2.md [hunk 2]
```diff
@@ -0,0 +1,164 @@
...
+
+$$E(x,y,z) = C(y, \text{Dec}(\text{Pred}(x), z)) + \lambda R(z)$$
+
+This method will limit the volume of space of z which takes a small value and the value which will, in turn, controls the space of y that has low energy. The value of Lambda controls this tradeoff. A useful example of $R$ is the $L_1$ norm, which can be viewed as an almost everywhere differentiable approximation of effective dimension. Adding noise to $z$ while limiting its $L_2$ norm can also limit its information content (VAE).
+
+### Sparse Coding
+
+Sparse coding is an example of an unconditional regularized latent-variable EBM which essentially attempts to approximate the data with a piecewise linear function.
+
+$$E(z, y) = ||y - Wz||^2 + \lambda ||z||_{L^1}$$
```
Comment (Atcold) [inline docs/en/week08/08-2.md]:
```suggestion
$$E(z, y) = \Vert y - Wz\Vert^2 + \lambda \Vert z\Vert_{L^1}$$
```

**Comment by kawshik8** [comment]
Should be fine now!

File: docs/en/week08/08-2.md [hunk 3]
```diff
@@ -0,0 +1,162 @@
...
+<center>
+<img src="{{site.baseurl}}/images/week08/08-2/fig2.png" width="90%"/><br>
+<b>Fig. 2</b>: FISTA computation graph
+</center>
+
+FISTA (fast ISTA) is an algorithm that optimizes the sparse coding energy function $E(y,z)$ with respect to $z$ by alternately optimizing the two terms $||y - Wz||^2$ and $\lambda ||z||_{L^1}$. We initialize $Z(0)$ and iteratively update $Z$ according to the following rule:
+
+$$z(t + 1) = \text{Shrinkage}_\frac{\lambda}{L}(z(t) - \frac{1}{L}W_d^T(W_dZ(t) - y))$$
+
+The inner expression $Z(t) - \frac{1}{L}W_d^T(W_dZ(t) - Y)$ is a gradient step for the $||y - Wz||^2$ term. The $\text{Shrinkage}$ function then shifts values towards 0, which optimizes the $\lambda ||z||_{L_1}$ term.
```
Comment (Atcold) [inline docs/en/week08/08-2.md]: Why is this math badly typed?! 😠😡
```suggestion
The inner expression $Z(t) - \frac{1}{L}W_d^\top(W_dZ(t) - Y)$ is a gradient step for the $\Vert y - Wz\Vert^2$ term. The $\text{Shrinkage}$ function then shifts values towards 0, which optimizes the $\lambda \Vert z\Vert_{L_1}$ term.
```

**Comment by Atcold** [comment]
You haven't updated the `_config.yml` file.
How is the website going to know you've submitted this?

**Comment by Atcold** [comment]
What is this garbage???????

Screenshot 2020-03-30 00 46 10

**Comment by kawshik8** [comment]
I'm checking it out

**Comment by Atcold** [comment]
Just fixed.

**Comment by Atcold** [comment]
More garbage.

Screenshot 2020-03-30 00 49 48

**Comment by Atcold** [comment]
Fixed.
Another this that's annoying is that the titles are all the same.

Screenshot 2020-03-30 00 45 26

**Pull request merged**